OracleDatabase/RAC/OracleRealApplicationClusters/dockerfiles/19.3.0/crontabEntry

```diff
@@ -1 +1,11 @@
-* * * * * /tmp/reset_failed_units.sh
+# LICENSE UPL 1.0
+#
+# Copyright (c) 1982-2019 Oracle and/or its affiliates. All rights reserved.
+#
+# Since: January, 2019
+# Author: sanjay.singh@oracle.com, paramdeep.saini@oracle.com
+# Description: Delete  a Grid node and add Oracle Database instance.
+#
+# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+
+* * * * * /tmp/resetFailedUnits.sh
```

Comment:
Can you explain why you're running this cronjob every minute of every day? Seems a bit excessive without context.